exist-core/src/main/java/org/exist/xquery/value/Type.java

```diff
@@ -105,95 +105,136 @@
     public final static int MAP = 102;
     public final static int ARRAY = 103;
     private final static Logger LOG = LogManager.getLogger(Type.class);
+
+    private static int NO_SUCH_VALUE = -99;
```

Comment:
If Type.NODE  value is changed to a positive value this should be set to -1 again.

Reply:
@adamretter ?